### PR TITLE
Single wind estimate from ekf selector

### PR DESCRIPTION
**Pull request by sfuhrer**
Fixes https://github.com/PX4/PX4-Autopilot/issues/16643

**Describe problem solved by this pull request**
- different sources of wind_estimate (for each airspeed sensor, from each ekf instance
- only first instance used, no failover if one ekf gets invalid

**Describe your solution**
Publish only single `wind_estimate` from currently used ekf instance. To this end:
- instead of publishing to `wind_estimate` in airspeed selector, publish to new `airspeed_selector_wind_estimate` (only for logging atm, not used anywhere outside of airspeed selector)
- each ekf instance publishes to `estimator_wind_estimate`, and ekf_selector then selects the one that's valid and publishes to `wind_estimate`

**Describe possible alternatives**

**Test data / coverage**

**Additional context**
- should we rename `wind_estimate` to `vehicle_wind`? or `vehicle_wind_estimate`? And `estimator_wind_estimate` to `estimator_wind`?

### CODE REVIEW

File: msg/airspeed_selector_wind_estimate.msg [hunk 1]
```diff
@@ -0,0 +1,22 @@
+uint64 timestamp		# time since system start (microseconds)
```
Comment (sfuhrer) [inline msg/airspeed_selector_wind_estimate.msg:1]: The alternative to adding a completely new topic would be to make an alias (wind_estimate, estimator_wind_estimate, airspeed_selector_wind_estimate). Downside: tas_scale and source is only used for the winds from airspeed selector.

Comment (dagar) [reply msg/airspeed_selector_wind_estimate.msg:1]: Keeping it separate is fine if it keeps things simple. Using the same .msg for multiple topics only saves a small amount of flash (logging fields and listener helpers).

**Comment by dagar** [comment]
This all looks correct. My only suggestion is to consider shortening the new topic names if anything else comes to mind.

 - `airspeed_selector_wind_estimateX` - would `airspeed_wind_estimateX` or `airspeed_selector_estimateX` make sense?
 - `estimator_wind_estimateX` seems a bit redundant

**Comment by sfuhrer** [comment]
> This all looks correct. My only suggestion is to consider shortening the new topic names if anything else comes to mind.
> 
>     * `airspeed_selector_wind_estimateX` - would `airspeed_wind_estimateX` or `airspeed_selector_estimateX` make sense?
> 
>     * `estimator_wind_estimateX` seems a bit redundant

Agree. What do you think of using `vehicle_wind`, `estimator_windX` and `airspeed_windX`?

**Comment by dagar** [comment]
> > This all looks correct. My only suggestion is to consider shortening the new topic names if anything else comes to mind.
> > ```
> > * `airspeed_selector_wind_estimateX` - would `airspeed_wind_estimateX` or `airspeed_selector_estimateX` make sense?
> > 
> > * `estimator_wind_estimateX` seems a bit redundant
> > ```
> 
> Agree. What do you think of using `vehicle_wind`, `estimator_windX` and `airspeed_windX`?

Sold.

I probably shouldn't even mention it, but I was thinking the whole `vehicle_` prefix has been a little over done and could probably be scaled back in the future.
  - vehicle_angular_velocity, vehicle_local_position, etc

**Comment by CarlOlsson** [comment]
> I probably shouldn't even mention it, but I was thinking the whole vehicle_ prefix has been a little over done and could probably be scaled back in the future.

+1 for that

**Comment by sfuhrer** [comment]
Alright, I've renamed 
- wind_estimate to vehicle_wind
- estimator_wind_estimate to estimator_wind
- airspeed_selector_wind_estimate to airspeed_wind

We can then separately look into removing the `vehicle_` prefix from all the messages if we wanna go that way (+1 from me as well).

**Comment by sfuhrer** [comment]
>We can then separately look into removing the vehicle_ prefix from all the messages if we wanna go that way (+1 from me as well).

On a second though, I've now already dropped the vehicle_ prefix for the wind topic. 684dc499ce1a7f753f33efa7397d2841cff96adf
Let me know your opinion.

**Comment by dagar** [comment]
> > We can then separately look into removing the vehicle_ prefix from all the messages if we wanna go that way (+1 from me as well).
> 
> On a second though, I've now already dropped the vehicle_ prefix for the wind topic. [684dc49](https://github.com/PX4/PX4-Autopilot/commit/684dc499ce1a7f753f33efa7397d2841cff96adf)
> Let me know your opinion.

We might have to rethink that if there's ever a reason to directly consume `wind` for some other reason (weather forecast), but it certainly seems like the simplest thing to do for now.

There's a small conflict with `msg/tools/uorb_rtps_message_ids.yaml`, but otherwise good to go.

**Comment by sfuhrer** [comment]
> There's a small conflict with msg/tools/uorb_rtps_message_ids.yaml, but otherwise good to go.

Okay, rebased on master.

**Comment by dagar** [comment]
Rebased again to resolve yet another rtps conflict.

**Review by dagar — APPROVED** [review]

**Pull request merged**